packages/cli/src/commands/destroy.js

```diff
@@ -5,7 +5,16 @@ import terminalLink from 'terminal-link'
 
 export const builder = (yargs) =>
   yargs
-    .commandDir('./destroy', { recurse: true })
+    .commandDir('./destroy', {
+      recurse: true,
+      /*
+      @NOTE This regex will ignore all double nested commands
+      e.g. /destroy/hi.js & destroy/hi/hi.js are picked up,
+      but destroy/hi/__tests__/whatever.test.js will be ignored
+      The [\/\\] bit is for supporting both windows and unix style paths
+      */
+      exclude: /destroy[\/\\]+.*[\/\\]+.*[\/\\]/,
+    })
```

Comment:
Hmmm you're right about the tests and fixtures directories because they get removed.... but for example, the new utils directory would be loaded, which is unnecessary. So would the templates folder!

Reply:
Bit more of an explanation here https://s.tape.sh/c3oHEnUC

Reply:
Tape for a tape: https://s.tape.sh/QSY3qBiy. Definitely watch, but here's a TL;DW:

- for destroy, `exclude` makes no difference, so I'd prefer to go without it
- for generate, `exclude` stops us from requiring one new file, `utils.js` as you said, but templates weren't loaded before and still aren't now because they don't end in .js (see extensions in this section: https://github.com/yargs/yargs/blob/main/docs/advanced.md#commanddirdirectory-opts)
- there's a difference between visiting files and loading them (i.e. requiring them); exclude only stops us from requiring them. As far as I know, commandDir doesn't have the luxury of not visiting files

I love rabbit holes, and this is pertinent to reworking the CLI anyway so it was worth going down, but I feel like we're bikeshedding a bit. (Or I am at least. 😅) I've spent a lot more time reviewing this change than the actual fix. That's my own fault—if I'm requested for a review, I try to understand every change, otherwise I don't feel like I'm doing my job. In this particular case, I kinda wish we could exclude this (pun not intended) and just focus on the CLI in another PR.

Reply:
All done, and good digging! We should definitely explore this more later